book/03-git-branching/sections/remote-branches.asc

```diff
@@ -7,12 +7,12 @@ You can get a full list of remote references explicitly with `git ls-remote [rem
 Nevertheless, a more common way is to take advantage of remote-tracking branches.
 
 Remote-tracking branches are references to the state of remote branches.
-They're local references that you can't move; they're moved automatically for you whenever you do any network communication.
-Remote-tracking branches act as bookmarks to remind you where the branches in your remote repositories were the last time you connected to them.
+They’re local references that you can’t move; Git moves them for you whenever you do any network communication, to make sure they accurately represent the state of the remote repository.
```

Comment:
Whoops, looks like GitHub did some smart quotes here. Let's let Asciidoctor manage those.

Reply:
Hmmmmm ... not sure how those got in there. I'll switch those two single quotes back and push.